docs/k0s-reset.md

```diff
@@ -0,0 +1,40 @@
+# k0s reset
+
+k0s Command-Line interface allows users to remove all k0s related files from the host.
+
+The reset command operates under assumption that k0s is running as a service on the host. and to prevent accidental triggering, it will not run if the k0s service is up. So first, we need to stop the service:
```

Comment:
```suggestion
The reset command operates under the assumption that k0s is installed as a service on the host. To prevent accidental triggering, it will not run if the k0s service is running. So first, we need to stop the service:
```